install-local-dev.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+# Download stable diffusion info
+cd backends/stable_diffusion_torch
+
+echo ">> Downloading stable diffusion code"
+# Download the m1-compatible Stable Diffusion sampling infrastructure
+git clone https://github.com/magnusviri/stable-diffusion.git
+cd stable-diffusion
+git checkout apple-mps-support
+cd ..
```

Comment:
Use `pushd` and `popd` instead of having to manually move around. (This script ends without returning the user to the root of the directory, by the by! With this, you can just `popd` at the end and leave the user at the beginning)

```suggestion
# Download stable diffusion info
pushd backends/stable_diffusion_torch

echo ">> Downloading stable diffusion code"
# Download the m1-compatible Stable Diffusion sampling infrastructure
git clone https://github.com/magnusviri/stable-diffusion.git
pushd stable-diffusion
git checkout apple-mps-support
popd
```